logstash-core-plugin-api/lib/logstash-core-plugin-api/version.rb

```diff
@@ -1,2 +1,2 @@
 # encoding: utf-8
-LOGSTASH_CORE_PLUGIN_API = "1.0.0"
+LOGSTASH_CORE_PLUGIN_API = "1.60.1"
```

Comment:
@andrewvc revert this back to 1.0.0 and then you can merge this in cleanly. You'd have to do a post PR for 2.4 to bump version to `1.60.1`